phpBB/assets/javascript/installer.js

```diff
@@ -13,6 +13,9 @@
 	var currentProgress = 0;
```

Comment:
@callumacrae if you could quickly estimate the damages, it would be much appreciated.